src/main/java/ru/curriculum/service/user/UserCRUDService.java

```diff
@@ -28,20 +29,41 @@
     }
 
     public UserDTO getUser(Integer userId) {
-        // TODO: либо применять get, либо проверять на null
         User user = userRepository.findOne(userId);
+        if(null == user) {
+            throw new UserNotFoundException(userId);
+        }
 
         return new UserDTO(user);
     }
 
-    public void create(UserDTO userDTO) {
-        User newUser = new User(userDTO);
+    public void create(UserDTO dto) {
+        User newUser = new User(
```

Comment:
Previous version is better.

Reply:
I wanted to gate away from transfer objects from upper layer to lower layer. Ok,  i'll try think of something else.